demo/secret/common/client.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"mosn.io/layotto/sdk/go-sdk/client"
+
+	runtimev1pb "mosn.io/layotto/spec/proto/runtime/v1"
+)
+
+var storeName string
+
+func init() {
+	flag.StringVar(&storeName, "s", "", "set `storeName`")
+}
+
+func main() {
+
+	flag.Parse()
+	if storeName == "" {
+		panic("storeName is empty.")
+	}
+	cli, err := client.NewClient()
+	if err != nil {
+		panic(err)
+	}
+	defer cli.Close()
+	ctx := context.Background()
+	//2. get the secret
+	resp, err := cli.GetSecret(ctx, &runtimev1pb.GetSecretRequest{
+		StoreName: storeName,
+		Key:       "db-user-pass:password",
+	})
+	fmt.Println(resp)
```

Comment:
Could you add some assertion here? Since we use these demos to run tests ,it's better to let the demo fail when something unexpected occurs.
For example:

```
if err!=nil {
  painc(err)
}
if resp ==nil || len(resp.Data)==0 {
  panic("no response data")
}
```


For more details ,see https://mosn.io/layotto/#/zh/development/test-quickstart?id=demo-%e4%bb%a3%e7%a0%81%e5%ba%94%e8%af%a5%e5%9c%a8%e4%b8%8d%e7%ac%a6%e5%90%88%e9%a2%84%e6%9c%9f%e6%97%b6-panic

Reply:
added